keyboards/converter/ps2_usb/keymaps/lbibass_at/rules.mk

```diff
@@ -0,0 +1,15 @@
+# Build Options
```

Comment:
This file can be deleted - it doesn't override anything in the keyboard-level rules.mk.